debezium-connector-cassandra/src/main/java/io/debezium/connector/cassandra/CommitLogProcessor.java

```diff
@@ -115,11 +121,14 @@ void processCommitLog(File file) throws IOException {
                 }
                 LOGGER.info("Successfully processed commit log {}", file.getName());
             }
-            catch (IOException e) {
+            catch (Exception e) {
                 if (!latestOnly) {
                     queue.enqueue(new EOFEvent(file, false));
                 }
                 LOGGER.warn("Error occurred while processing commit log " + file.getName(), e);
+                if (commitLogTransfer.getClass().toString() == CassandraConnectorConfig.DEFAULT_COMMIT_LOG_TRANSFER_CLASS) {
```

Comment:
Do we really want to use equality operator here? Also `DEFAULT_COMMIT_LOG_TRANSFER_CLASS` contians the name of the calss but the `toString` here is called the `class` itself.

Reply:
Nice catch! Correct codes accordingly :)

Reply:
> Do we really want to use equality operator here? Also `DEFAULT_COMMIT_LOG_TRANSFER_CLASS` contians the name of the calss but the `toString` here is called the `class` itself.

Nice catch! Correct codes accordingly :)